bin/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash 
+#!/bin/bash -e
```

Comment:
what does the `-e` do?

Reply:
I makes the shell script exit on error rather than proceeding to the next line.

```   -e  Exit immediately if a simple command exits with a non-zero status, unless
       the command that fails is part of an until or  while loop, part of an
       if statement, part of a && or || list, or if the command's return status
       is being inverted using !.  -o errexit
```

Reply:
Thanks Toby, good to know. I often used `command || exit 1` but the `-e` flag seems like the better approach (in a build script).